addons/spreadsheet/static/src/list/plugins/list_core_plugin.js

```diff
@@ -90,6 +98,12 @@ export default class ListCorePlugin extends CorePlugin {
                 this.history.update("nextId", parseInt(id, 10) + 1);
                 break;
             }
+            case "DUPLICATE_ODOO_LIST": {
+                const { listId, newListId } = cmd;
+                this._addList(newListId, this.lists[listId].definition);
```

Comment:
```suggestion
                this._addList(newListId, deepCopy(this.lists[listId].definition));
```
you pass the reference, if I update the domain of the first DS, it will affect the second one as well